src/libsyncengine/syncpal/syncpal.h

```diff
@@ -202,9 +202,7 @@ class SYNCENGINE_EXPORT SyncPal : public std::enable_shared_from_this<SyncPal> {
         ExitCode cancelDlDirectJobs(const std::list<SyncPath> &fileList);
         ExitCode cancelAllDlDirectJobs(bool quit);
         ExitCode cleanOldUploadSessionTokens();
-        inline bool isDlOngoing(const SyncPath &localPath) {
-            return _syncPathToDownloadJobMap.find(localPath) != _syncPathToDownloadJobMap.end();
-        }
+        bool isDlOngoing(const SyncPath &localPath);
```

Comment:
Would it be possible to find a more suggestive name? Does `Dl` stand for `Download`?

Reply:
Yes. Renamed.